src/widgets/course-main/ui/course-main.tsx

```diff
@@ -14,11 +15,13 @@ import { WidgetTitle } from '@/shared/ui/widget-title';
 
 import styles from './course-main.module.scss';
 
-interface CourseMainProps {
+const cx = classNames.bind(styles);
+
+export type CourseMainProps = {
   courseName: string;
   lang?: 'ru' | 'en';
   type?: 'Pre-school RU';
-}
+};
 
 const localizedContent = {
```

Comment:
Rename `localizedContent` and move it to the `dev-data` folder

Reply:
What is the `dev-data` folder? Is it an FSD segment or something else?

Reply:
> Rename `localizedContent`

Rename it to what?

Reply:
Done